spec/abstractops.html

```diff
@@ -1499,6 +1499,21 @@ <h1>ToIntegerThrowOnInfinity ( _argument_ )</h1>
     </emu-alg>
   </emu-clause>
 
+  <emu-clause id="sec-temporal-tointegernotruncate" aoid="ToIntegerNoTruncate">
+    <h1>ToIntegerNoTruncate ( _argument_ )</h1>
```

Comment:
this is kind of a weird name, since "to x" usually coerces or throws, and this is more like "ToNumber, but throw if it's not an integer". i don't have a better suggestion atm tho :-/

Reply:
It does return ℝ(_number_), so ToInteger seems appropriate, but I agree that NoTruncate could be replaced by something better, maybe in the same vein as ToIntegerThrowOnInfinity.  (ToIntegerThrowOnNonInteger seems like it would be very unfortunate though.) Let me know if you have any ideas.

Reply:
Maybe ToIntegerWithoutRounding?

Reply:
Works for me!